### ISSUE TITLE
Remove or move roadmap?

### ISSUE THREAD

**Issue by matthewmayer** [faker-js/faker#2666]
The roadmap at https://fakerjs.dev/about/roadmap/ is quite outdated. It says various things may happen in v8 which will actually now happen in v9. 

Now we could easily update this and add a v9 section but I feel in the long run having the roadmap directly on the docs website isn't ideal, as you will end up with multiple versions snapshotted in time:

https://next.fakerjs.dev/about/roadmap/
https://fakerjs.dev/about/roadmap/
https://v8.fakerjs.dev/about/roadmap/
https://v7.fakerjs.dev/about/roadmap/

Maybe it would be better to have a stub (like https://next.fakerjs.dev/about/contributing.html) which directly links to a canonical ROADMAP.md directly at the repo root which would be easier to keep updated?

**Comment by Shinigami92** [comment]
If we would like to boost it to real roadmap, we could also tryout GitHub's Project-Roadmap feature.
Last time we introduced Project Boards, this Roadmap feature was not available.

=> https://github.blog/changelog/2023-03-23-roadmaps-in-projects-are-now-generally-available/

**Comment by matthewmayer** [comment]
I think I prefer something in markdown where you can list out high level goals in plain English . Not having to link lots of tickets, add specific dates etc.

**Comment by Shinigami92** [comment]
> I think I prefer something in markdown where you can list out high level goals in plain English . Not having to link lots of tickets, add specific dates etc.

Understandable :ok_hand:  

So yeah, a project board roadmap might more be something that is useful for internally managing, but not for representing progress to community.
But I also thing we dont really need a internal roadmap then, cause managing the start and due dates for issues and PRs are **very** uncertain.

**Comment by xDivisionByZerox** [comment]
> Maybe it would be better to have a stub (like [next.fakerjs.dev/about/contributing.html](https://next.fakerjs.dev/about/contributing.html)) which directly links to a canonical ROADMAP.md directly at the repo root which would be easier to keep updated?

How is that different from having a docs page where you are writing markdown as well?

**Comment by matthewmayer** [comment]
Because you don't have multiple old versions hanging about on the docs website

**Comment by matthewmayer** [comment]
Maybe the simplest would be to stub out the v8 roadmap so that

https://fakerjs.dev/about/roadmap/ just has

`For the latest roadmap, see https://next.fakerjs.dev/about/roadmap/`

**Comment by ST-DDT** [comment]
**Team Proposal**

We should make a more active usage of the roadmap.
Every time before we do a release, we plan what we want to do for the next release and document that in the roadmap file.
This ensures everyone has the same expectations for the upcoming version.
This guarantees that the roadmap is up to date for the next release.
Then there is no need to keep it in GH as the website has the latest version on release.
For v8 we will do a one time backport of the roadmap.

**Comment by xDivisionByZerox** [comment]
@matthewmayer Do you have any disagreements with our previously made proposal? Otherwise, I'd close this issue.

**Comment by matthewmayer** [comment]
No sounds good to me. My root concern was that it was outdated, so updating it more frequently solves this problem.
